controller/deis/settings.py

```diff
@@ -136,6 +140,8 @@
     'django.contrib.sites',
     'django.contrib.staticfiles',
     # Third-party apps
+    'django_auth_ldap',
+    'django_fsm',
```

Comment:
I'm not a django expert, some people said that would be better if we put the module here, some people don't. For safe precautions I prefer to have it explicit. I can remove if you think that its better. :+1: